src/ameba/rule/lint/missing_block_argument.cr

```diff
@@ -36,10 +34,7 @@ module Ameba::Rule::Lint
     def test(source, node : Crystal::Def, scope : AST::Scope)
       return if !scope.yields? || node.block_arg
 
-      return unless location = node.name_location
-      end_location = name_end_location(node)
-
-      issue_for location, end_location, MSG
+      issue_for node, MSG, prefer_name_location: true
```

Comment:
does this change the behaviour? Previously if name_location was empty it returned but now it uses a node's location as a fallback?

Reply:
Indeed, that was a fix on my part, since the end location is not required for an issue.